modules/claims_api/config/schemas/526.json

```diff
@@ -542,6 +552,9 @@
                 },
                 "serviceRelevance": {
                   "type": "string"
+                },
+                "specialIssues": {
```

Comment:
Why does this exist twice in the schema?

Reply:
See: 

https://github.com/department-of-veterans-affairs/vets-api/pull/4931#discussion_r491197426

Reply:
Good job.